R/helpers.R

```diff
@@ -6,9 +6,10 @@ model_file_dict <- function(run_id = NULL, year = NULL) {
   suppressPackageStartupMessages(library(magrittr))
 
   # Convert flat dictionary file to nested list
-  dict <- readr::read_csv(
+  dict <- read.csv(
     here::here("misc", "file_dict.csv"),
-    col_types = readr::cols()
+    colClasses = c("character", "character", "numeric", rep("character", 9)),
+    na.strings = ""
```

Comment:
Why the switch here? `readr` should be in the dependencies IIRC

Reply:
This is a syntax copied from the res repo